src/external-api/data/vessel_list.json

```diff
@@ -0,0 +1,24 @@
+{
+  "vessel_list": [
+    {
+      "id": "1",
+      "name": "SS_Martha",
+      "company_id": "0"
```

Comment:
I just noticed that we do not have any information about the companies. Can you please create another json file with some fake company names (no need to ask Optimum Voyage to output real customers and then obscure the names after).

I guess we only need 4 company names/ids with the current data, but if you want to go down the easy route you could use https://mockaroo.com/ to generate some fake data :)